src/IECore/StreamIndexedIO.cpp

```diff
@@ -362,6 +362,33 @@ class StreamIndexedIO::StringCache
 namespace
 {
 
+const static std::map<std::string, int> nameCodeMapping = {{"blosclz", 0}, {"lz4", 1}, {"lz4hc", 2}, {"snappy", 3}, {"zlib", 4}};
+
+//! map blosc compressor name to a int which we can serialise into
+//! the indexedIO header. We don't use the blosc header defined values incase they change.
+int getCompressionCode( const std::string &compressor )
```

Comment:
Good idea not using the blosc values directly, but can we make our own public enum for these option values? Or do you think that's tying our hands a bit?